modules/invibesBidAdapter.js

```diff
@@ -0,0 +1,438 @@
+import * as utils from 'src/utils';
+import { registerBidder } from 'src/adapters/bidderFactory';
+
+const CONSTANTS = {
+  BIDDER_CODE: 'invibes',
+  BID_ENDPOINT: '//bid.videostep.com/Bid/VideoAdContent',
+  SYNC_ENDPOINT: '//k.r66net.com/GetUserSync',
+  TIME_TO_LIVE: 300,
+  DEFAULT_CURRENCY: 'EUR',
+  PREBID_VERSION: 1,
+  METHOD: 'GET'
+};
+
+export const spec = {
+  code: CONSTANTS.BIDDER_CODE,
+  aliases: [CONSTANTS.BIDDER_CODE],
```

Comment:
Sorry, I don't understand where the mistake is. If I don't export, my tests are failing because I can't import the spec. If I don't register the spec, my bidder is not called

Reply:
@rcheptanariu I should have explained it in detail. 
By adding `aliases: [CONSTANTS.BIDDER_CODE]`, you are re-registering the adapter as you are using the same constant.  
So you should remove the line `aliases: [CONSTANTS.BIDDER_CODE]` or use some different alias.